index.html

```diff
@@ -15,14 +16,28 @@
         <section class="header">
             <header>
                 <nav class="normal_button">
-                    <img class="menu-icon" src="images/Icon - Menu.png" width="24px" height="24px" alt="Menu-Icon">
+
                     <ul class="desktop-menu">
-                        <li><a href="#">Portfolio</a></li>
-                        <li><a href="#">About</a></li>
-                        <li><a href="#">Contact</a></li>
+                        <li><a href="#Portfolio">Portfolio</a></li>
+                        <li><a href="#About">About</a></li>
+                        <li><a href="#Contact">Contact</a></li>
```

Comment:
- [ ] Another issue in your mobile-menu is that when user click on hamburger button, the rest of the page is still open to the user and he can scroll as well, this is something against the design as we can see in the figma template. you can achieve that by applying extra CSS given below by me and also you need to add `overflow-y: hidden;` to `body` when user clicks on hamburger, with javascript and you would revert that styling when user clicks on cross or any other link again.
css given by me
![a](https://github.com/batoolfatima2135/Portfolio/assets/96683843/68e60f58-8ea9-4cc1-9794-668d234e1cc4)

figma design
![qw](https://github.com/batoolfatima2135/Portfolio/assets/96683843/a2cfee3f-d18c-4c53-88f8-1c45d797f294)